### PR TITLE
Buckle Tweak

### PR DESCRIPTION
**Pull request by NanakoAC**
changes:
  - bugfix: "You can no longer buckle people into a chair/bed/etc which is already occupied."
  - tweak: "Buckling yourself into something is no longer a visible action, only you will see the message."

The former change, while working as intended, is causing undesireable behaviour, and i think this way makes better sense.

The last one is a brief idea i had to reduce noise in the chatlog. If anyone wants to see if you're buckled in, they can examine you. A message in the chatlog is needless noise and not relevant information to most people. It's most noticeable on the transfer shuttle where the large number of people buckling can impede communication

People buckling others in or unbuckling is still visible

### CODE REVIEW

**Comment by NanakoAC** [comment]
Fixes #737 and #736

File: code/game/objects/buckling.dm [hunk 1]
```diff
@@ -61,16 +61,16 @@
...
-	unbuckle_mob()
+	unbuckle_mob()//this is now just for safety, buckling someone into an occupied chair will fail, instead of removing the occupant
 
 	if(buckle_mob(M))
 		if(M == user)
-			M.visible_message(\
-				"<span class='notice'>[M.name] buckles themselves to [src].</span>",\
-				"<span class='notice'>You buckle yourself to [src].</span>",\
-				"<span class='notice'>You hear metal clanking.</span>")
+			M << "<span class='notice'>You buckle yourself to [src].</span>"
```
Comment (skull132) [inline code/game/objects/buckling.dm]: This change doesn't make sense?

It makes sense that someone buckling themselves onto a chair/bed/whatever sends a visible message.

**Comment by skull132** [comment]
@NanakoAC Left a note to address. I don't see the need to make buckling yourself a thing visible only to the user themselves.

**Comment by NanakoAC** [comment]
It's about signal to noise ratio.

Yes you're right, it does make sense. But in most situations, it's not relevant information, 90% of the time you don't need to know that someone is buckling themselves in, and the message is just a tiny annoyance.

As mentioned, if for any reason you do need to check if someone is buckled (maybe to figure out whether or not you can drag them, or push them over) you can examine them to get that information. 

On the shuttle, it just becomes a spam of [X buckles themselves to the chair] filling up the chatlog.

**Comment by skull132** [comment]
The shuttle spam is unfortunate, but not a reason to remove this information.

There is no other visual indicator of someone having buckled themselves, and it can play a role in the present. Removing it simply because of _one_ instance is unnecessary.

Please revert that specific change, and it'll be good to go.

**Comment by NanakoAC** [comment]
that change is reverted

**Pull request merged**